.github/workflows/pg_dump-restore-test.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           install_dir: ${{ steps.find-branch.outputs.base-dir }}
           migration_mode: 'multi-db'
           dump_restore: 'true'
-          antlr_version: ${{ (steps.read-base-and-final-version.outputs.base-version == 'source.latest' || fromJSON(steps.read-base-and-final-version.outputs.base-version) >= 16.8 ) && '4.13.2' || '4.9.3' }}
+          antlr_version: ${{ (steps.read-base-and-final-version.outputs.base-version == 'source.latest' || fromJSON(steps.read-base-and-final-version.outputs.base-version) >= 16.8 || steps.read-base-and-final-version.outputs.base-version == '16.10') && '4.13.2' || '4.9.3' }}
```

Comment:
Perhaps we need a better mechanism to make this comparison work. Ideally there shouldn't be a need to add this additional comparison for 16.10 if >= 16.8 is already there.

Reply:
agreed, although because this is a yaml qualifier and not in bash I'm not sure if there's a quick way to fix this. I think we should file a JIRA as a followup to improve this process in general.